docs/development.md

```
@@ -44,11 +44,11 @@ environment.
 
    - Install kea, but might be out of date:
      ```
-     sudo apt-get update && sudo apt-get install -y isc-kea-dhcp4-server isc-kea-dev
+     sudo apt-get update && sudo apt-get install -y isc-kea-dhcp4-server isc-kea-dhcp6-server isc-kea-dev
      ```
    - Or, but has only been tested with Ubuntu 23.10, install kea:
```

markdownlint-cli2 (0.23.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 4; Actual: 3

(MD005, list-indent)